Add documentation about how to deal with RHUI (#229)
uyuni-project/uyuni-docs: modules/client-configuration/pages/clients-rh.adoc

To manage repositories on your {redhat} client, you need to import the CA and entitlement certificates to the {productname} Server.
This requires three entries: one each for the entitlement certificate, the entitlement key, and the {redhat} certificate.

[NOTE]
====

> If your {redhat} client is using Red Hat Update Infrastructure (RHUI) repositories, you would need to use a different set CA and entitlement certificates. As an example:

Reviewer: ```suggestion
If your {redhat} client is using Red Hat Update Infrastructure (RHUI) repositories, you need to use a different set of CA and entitlement certificates. 
For example:
```

. Click btn:[Sync Now] to begin synchronization immediately.

[IMPORTANT]
====

> In order to allow [command]``spacewalk-repo-sync`` to sync content from RHUI repositories, we need to manually add the required [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` required HTTP headers to [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` file:

Reviewer: ```suggestion
For RHUI repositories, you must manually add the [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` required HTTP headers to the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file.
Without these headers, the [command]``spacewalk-repo-sync`` command will not be able to synchronize content from the RHUI repositories.
```

----
[channel_label]
X-RHUI-ID=value
X-RHUI-SIGNATURE=value
----

> An example about how to calculate those values for the required HTTP headers from a {rhela} instance running on the Public Cloud:

Reviewer: ```suggestion
Use this example to calculate the the required values for the HTTP headers from a {rhel} instance running on a public cloud:
```

. OPTIONAL: Navigate to the [guimenu]``Sync`` tab to set a recurring schedule for synchronization of this repository.
. Click btn:[Sync Now] to begin synchronization immediately.

[IMPORTANT]
====
In order to allow [command]``spacewalk-repo-sync`` to sync content from RHUI repositories, we need to manually add the required [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` required HTTP headers to [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` file:

----
[channel_label]
X-RHUI-ID=value
X-RHUI-SIGNATURE=value
----

An example about how to calculate those values for the required HTTP headers from a {rhela} instance running on the Public Cloud:

----
echo "X-RHUI-ID=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|base64|tr -d '\n')"
echo "X-RHUI-SIGNATURE=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/signature|base64|tr -d '\n')"

> ----

Reviewer: See my edits above :point_up:

Author: Done. Thanks!
